data_steward/cdr_cleaner/cleaning_rules/ehr_submission_data_cutoff.py

```diff
@@ -0,0 +1,240 @@
+"""
+Enforces a data cutoff date for PPI data.
+
+Original Issue: DC-1445
+
+Intent is to enforce the data cutoff date for PPI data in all CDM tables excluding the person table by sandboxing and
+    removing any records that persist after the data cutoff date.
+"""
+
+# Python imports
+import logging
+from datetime import datetime
+
+# Project imports
+from common import JINJA_ENV
+from utils import pipeline_logging
+from constants import bq_utils as bq_consts
+from resources import fields_for, CDM_TABLES
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+LOGGER = logging.getLogger(__name__)
+
+SANDBOX_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{intermediary_table}}` AS (
+SELECT * FROM `{{project_id}}.{{dataset_id}}.{{cdm_table}}`
+WHERE 
+    (coalesce({{date_fields}}) < DATE("{{cutoff_date}}")) 
+    AND coalesce({{datetime_fields}}) < TIMESTAMP("{{cutoff_date}}"))
+""")
+
+DATE_CUTOFF_QUERY = JINJA_ENV.from_string("""
+SELECT * FROM `{{project_id}}.{{dataset_id}}.{{cdm_table}}` cdm
+EXCEPT DISTINCT
+SELECT * FROM `{{project_id}}.{{sandbox_id}}.{{intermediary_table}}`
+""")
+
+
+class EhrSubmissionDataCutoff(BaseCleaningRule):
+    """
+    All rows of data in the RDR ETL with dates after the cutoff date should be sandboxed and dropped
+    """
+
+    def __init__(self,
+                 project_id,
+                 dataset_id,
+                 sandbox_dataset_id,
+                 cutoff_date=None):
+        """
+        Initialize the class with proper information.
+
+        Set the issue numbers, description and affected datasets. As other tickets may affect
+        this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+
+        :params: truncation_date: the last date that should be included in the
```

Comment:
Changing docstring param name from `truncation_date` to `cutoff_date`
```suggestion
        :params: cutoff_date: the last date that should be included in the
```